lib/generators/rodauth/templates/app/views/rodauth/tailwind/close_account.html.erb

```diff
@@ -0,0 +1,23 @@
+<% content_for :title, rodauth.close_account_page_title %>
+<div class="dark:bg-gray-900 flex items-center justify-center h-screen">
+  <div class="mx-4 py-4">
+    <h3 class="mb-3 text-3xl font-semibold text-center text-white"> Close Account </h3>
+    <div class="flex p-4 mb-4 text-sm text-red-700 bg-red-100 rounded-lg dark:bg-red-200 dark:text-red-800" role="alert">
+      <div>
+        <span class="font-medium">Warning!</span> You're about to close your account!
+      </div>
+    </div>
```

Comment:
While I see this being useful, I would prefer we don't include it in the default template, since it won't be translated.

Reply:
> While I see this being useful, I would prefer we don't include it in the default template, since it won't be translated.

Given the translation concerns, I have commented out the text, allowing users to implement what they feel is best, because I thought the gravity of some type of warning necessary. If this is unsuitable, please LMK and I will immediate omit it altogether: [76f8fae](https://github.com/janko/rodauth-rails/pull/114/commits/76f8fae42b65d3a508f7225cea26caaf995e2208)